stdlib/universe/table_fns_test.go

```diff
@@ -346,6 +348,8 @@ t = inj |> tableFind(fn: (key) => key.user == "user1")`
 // We have to write this test as a non-standard e2e test, because
 // our framework doesn't allow comparison between "simple" values, but only streams of tables.
 func TestIndexFns_EndToEnd(t *testing.T) {
+	t.Skip("table cannot be used more than once")
```

Comment:
I should file an issue for this because it breaks tableFind/getColumn/GetRecord use.

Reply:
Yes. I can help you with this. I think we will have to have `objects.NewTable` call `CopyTable` and then we will have to find a way to have a scope call `Done` on tables after they go out of scope so we can release the values.